src/brokers/ebase.js

```diff
@@ -124,8 +142,12 @@ export const canParsePage = (content, extension) =>
   content.some(line => line.includes('Fondsertrag / Vorabpauschale'));
 
 export const parsePages = contents => {
+  const activities = parseData(contents);
+  const status = parseData(contents) !== undefined ? 0 : 6
```

Comment:
We can reduce the duplicated work to once
```suggestion
  const status = activities !== undefined ? 0 : 6
```

Reply:
Absolutely. Brainfart of mine